mergify_engine/tests/functional/test_engine_v2.py

```diff
@@ -1395,6 +1395,33 @@ def test_refresh_on_conflict(self):
             "issue_comment", {"action": "created", "comment": {"body": "It conflict!"}},
         )
 
+    def test_command_update(self):
+        rules = {
+            "pull_request_rules": [
+                {
+                    "name": "auto-rebase-on-conflict",
+                    "conditions": ["conflict"],
+                    "actions": {"comment": {"message": "nothing"}},
+                }
+            ]
+        }
+        self.setup_repo(yaml.dump(rules), files={"TESTING": "foobar"})
+        p1, _ = self.create_pr(files={"TESTING2": "foobar"})
+        p2, _ = self.create_pr(files={"TESTING3": "foobar"})
+        p1.merge()
+
+        self.wait_for("pull_request", {"action": "closed"})
+
+        self.create_message(p2, "@mergifyio update")
+
+        if base.RECORD:
+            time.sleep(1)
```

Comment:
looks a bit… fuzzy but ok I guess?

Reply:
I copy/paste the rebase test, but it's not needed here.